docs/getting-started/installation.md

```diff
@@ -41,23 +41,23 @@ kubectl port-forward -n kusk-system svc/kusk-gateway-private-envoy-fleet 8080:80
 open http://localhost:8080
 ```
 
-## Get the Gateway's External IP
+## **Get the Gateway's External IP**
 
-If you want to access the APIs or StaticRoutes managed by Kusk Gateway you can get the External IP address of the 
-Load Balancer by running the command below command. Note that it may take a few seconds for the LoadBalancer IP to become available.
+If you want to access the APIs or StaticRoutes managed by Kusk Gateway, get the External IP address of the 
+Load Balancer by running the command below. Note that it may take a few seconds for the LoadBalancer IP to become available.
 
 ```sh
 kubectl get svc -l "app.kubernetes.io/component=envoy-svc" --namespace kusk-system
 ```
 
 The output should contain the [Envoy Fleet](../customresources/envoyfleet) Service, which is the entry point of your API gateway, with the **External-IP** address field - use this address for your API endpoints querying. Note that it might take a while for the External IP to be created.
 
-!!! note non-important "External IP might not be available for some cluster setups"
+**Note:** Non-important "External IP might not be available for some cluster setups".
```

Comment:
```suggestion
!!! note non-important "External IP might not be available for some cluster setups".
```

Unfortunately, this has to stay as it was because these words are mk-docs specific configurations